server.js

```diff
@@ -49,14 +49,32 @@ monitor.get("/recent-events", async (req, res) => {
 const webhooks = express();
 const webhooksPort = config.port + 1;
 
-webhooks.use(bodyParser.json());
 webhooks.listen(webhooksPort, () => {
   console.log(`Listening for webhooks: http://localhost:${webhooksPort}`);
 });
 
 // Provides an endpoint to receive webhooks
-webhooks.post("/", async (req, res) => {
-  let event = req.body;
+webhooks.post("/", bodyParser.raw({ type: "application/json" }), async (req, res) => {
```

Comment:
Curious: why do we need to follow this pattern rather than simply using the middle ware for Express? (line removed in PR: `webhooks.use(bodyParser.json())`)

Reply:
Without this, it fails to marshal the secret properly:

```
Failed to verify webhook signature: No signatures found matching the expected signature for payload. Are you passing the raw request body you received from Stripe? https://github.com/stripe/stripe-node#webhook-signing
```